src/Execution/ValuesHelper.php

```diff
@@ -259,8 +258,8 @@ private function coerceValue($value, $type, $blameNode, ?Path $path = null): Coe
             return $this->coerceValueForNonNullType($value, $type, $blameNode, $path);
         }
 
-        if (empty($value)) {
-            return new CoercedValue(null, null);
+        if (null === $value) {
```

Comment:
👍

Reply:
I thought we fix this already :-?

Reply:
Apparently we didn't or maybe it got unfixed at some point. We did have unit tests for this, but `assertEquals` apparently does a loose comparison of array values so the tests didn't fail even though they should have.

Reply:
OK, got it.